tests/Resources/Files/kubeflow-operator.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+   name: data-science-pipelines-operator
```

Comment:
This file should be called data-science-pipelines-operator.yaml, not kubeflow-operator.yaml